styleguide.config.js

```diff
@@ -33,6 +33,11 @@ module.exports = {
                 "**/components/modal-launcher-portal.js",
             ],
         },
+        {
+            name: "Button",
+            content: "packages/wonder-blocks-button/docs.md",
+            components: "packages/wonder-blocks-button/components/*.js",
```

Comment:
You should ignore button-core.js (like we did for some of the modal files, above) as we aren't expecting the user to ever interact with it directly.